lms/static/js/learner_dashboard/views/program_details_view.js

```diff
@@ -31,6 +35,16 @@
 
                 postRender: function() {
                     // Add subviews
+                    new CollectionListView({
+                        el: '.js-course-list',
+                        childView: CourseCardView,
+                        collection: new Backbone.Collection(this.context.course_codes),
```

Comment:
I think now it definitely the time to address this, while we are blocked on design we should ensure the functionality is how we want it. It is odd that the collection is not being recognized in the postRender function if defined in initialize, I would advocate for debugging why that is the case. The initialize function is there to set things up before rendering the DOM so it is worrying that this is not working here.